src/dkg/key_generation.rs

```diff
@@ -1196,6 +1196,197 @@ mod test {
     use rand::rngs::OsRng;
     use rand::Rng;
 
+    pub(crate) fn do_keygen(
+        n1: u32,
+        t1: u32,
+        n2: Option<u32>,
+        t2: Option<u32>,
+    ) -> FrostResult<
+        Secp256k1Sha256,
+        (
+            ThresholdParameters<Secp256k1Sha256>,
+            Vec<IndividualSigningKey<Secp256k1Sha256>>,
+            GroupVerifyingKey<Secp256k1Sha256>,
+            Option<ThresholdParameters<Secp256k1Sha256>>,
+            Option<Vec<IndividualSigningKey<Secp256k1Sha256>>>,
+        ),
+    > {
+        type Dkg<T> = DistributedKeyGeneration<T, Secp256k1Sha256>;
+
+        let params = ThresholdParameters::new(n1, t1);
+        let rng = OsRng;
+
+        let mut participants = Vec::<Participant<Secp256k1Sha256>>::new();
```

Comment:
Nit: We can use `with_capacity` (similarly for the other `new`s in the function).

Reply:
Yeah, though I've never bothered with these in the past as they are within tests only